src/endpoint/s3/s3_bucket_policy_utils.js

```diff
@@ -281,6 +281,17 @@ async function validate_s3_policy(policy, bucket_name, get_account_handler) {
     }
 }
 
+/*
+    CRUD action on bucket policy only allow to bucket owner 
+*/
+async function validate_s3_policy_owner(bucket_owner_id, account) {
+    if (bucket_owner_id !== account._id) {
+        const err_msg = 'The specified method is not allowed against this resource.';
```

Comment:
I think you can remove this `err_msg`, the RPC call `METHOD_NOT_ALLOWED` is mapped to `S3Error.MethodNotAllowed` and you have this message inside, see:
https://github.com/noobaa/noobaa-core/blob/bfaa477af56a995a6fc0238f5a41e668f43840d6/src/endpoint/s3/s3_errors.js#L612
and:
https://github.com/noobaa/noobaa-core/blob/bfaa477af56a995a6fc0238f5a41e668f43840d6/src/endpoint/s3/s3_errors.js#L292-L296